root/report/ArtistsThatMayBeGroups.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+import formatUserDate from '../utility/formatUserDate';
+import {l} from '../static/scripts/common/i18n';
+
+import ArtistList from './components/ArtistList';
+import FilterLink from './FilterLink';
+import type {ReportArtistT, ReportDataT} from './types';
+
+const ArtistsThatMayBeGroups = ({
+  $c,
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportArtistT>) => (
+  <Layout fullWidth title={l('Artists that may be groups')}>
+    <h1>{l('Artists that may be groups')}</h1>
+
+    <ul>
+      <li>{l('This report lists artists that have type set to other than Group (or a subtype of Group) \
```

Comment:
While you're changing the string, s/have type set/have their type set/ sounds more Englisher. Same in ArtistsThatMayBePersons